config/job_types/ingest_book.yml

```diff
@@ -3,7 +3,7 @@ tasks:
   - foreach: "*.[tT][iI][fF]*"
     do: convert_tif_to_jpg
   - foreach: "*.[jJ][pP][gG]*"
-    do: jpg_to_pdf
-  - pdf_merge_converted
+    do: convert_jpg_to_pdf
+  - merge_converted_pdfs
```

Comment:
Vorschlag: Task umbenennen in `merge_pdf`. Theoretisch kann der Task ja auch an anderer Stelle verwendet werden, um PDFs zu mergen, die nicht konvertiert wurden.